extensions/objects/models/objects_test.py

```diff
@@ -1318,3 +1319,19 @@ def test_normalize_value(self):
         self.assertEqual(
             objects.TranslatableSetOfUnicodeString.normalize_value(['1', '2']),
             ['1', '2'])
+
+
+class JsonEncodedInStringTests(test_utils.GenericTestBase):
+
+    def test_normalization(self):
+        list_of_ids = ['0', '1']
+        with self.assertRaisesRegexp(
+            Exception, 'Expected string received 2'
```

Comment:
```suggestion
            Exception, 'Expected string received 2 of type int'
```

Reply:
Why did the backend tests not fail when you added the type in the error message?

Reply:
I have tested for other methods and found out that ```assertRaisesRegexp``` do not raises exception if some parts of the string matched.
For reference please take a look here: https://github.com/oppia/oppia/blob/6fb20d0175ce401603bbc8b6cf8b75be24ccd40a/core/domain/object_registry_test.py#L37

Reply:
Updated.